tools/integration_tests/gzip/gzip_test.go

```diff
@@ -172,6 +180,46 @@ func destroy_testdata(m *testing.M) error {
 	return nil
 }
 
+// createDataOfSize generates a string of the specified content size in bytes.
+// Failure cases:
+// 1. contentSize <= 0
+func createDataOfSize(contentSize int) (string, error) {
```

Comment:
nit: for consistency

```suggestion
func createContentOfSize(contentSize int) (string, error) {
```